scripts/travis-test.sh

```diff
@@ -12,8 +12,10 @@ echo 'Running tests in reverse...'
 echo 'Linting...'
 ./scripts/lint.sh
 
-echo 'Running seltests...'
-./scripts/travis-run-pdiff-tests.sh
+# Seltests are disabled due to inconsistent screenshots 
```

Comment:
I'd just kill these lines entirely; we can always get them back via source control!